network-discovery/policy/manager.go

```diff
@@ -0,0 +1,94 @@
+package policy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+
+	"github.com/netboxlabs/diode-sdk-go/diode"
+	"gopkg.in/yaml.v3"
+
+	"github.com/netboxlabs/orb-discovery/network-discovery/config"
+)
+
+// Manager represents the policy manager
+type Manager struct {
+	policies map[string]*Runner
+	client   diode.Client
+	logger   *slog.Logger
+	ctx      context.Context
+}
+
+// Configure configures the policy manager
+func (m *Manager) Configure(ctx context.Context, logger *slog.Logger, client diode.Client) error {
+	m.ctx = ctx
+	m.logger = logger
+	m.client = client
+	m.policies = make(map[string]*Runner)
+	return nil
+}
```

Comment:
It would be more idiomatic to add `NewManager()` instantiating `Manager` instead and no error is ever returned so redundant as well, example:

```suggestion
// NewManager instantiates a new policy manager
func NewManager(ctx context.Context, logger *slog.Logger, client diode.Client) *Manager {
    return &policy.Manager{
        ctx:      ctx,
        client:   client,
        policies: make(map[string]*Runner),
    }
}
```

Reply:
cool! did that